CollAction/Views/Account/ExternalLogin.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@using Microsoft.AspNetCore.Mvc.Localization
+
+@model ExternalLoginViewModel
+
+@inject IViewLocalizer Localizer
+
+@{
+    ViewData["Title"] = "Register";
+}
+
+<div id="login-starting">
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <h1>@ViewData["Title"]</h1>
+                <h4>Associate your @ViewData["LoginProvider"] account.</h4>
+                <p class="text-info">
+                    You've successfully authenticated with <strong>@ViewData["LoginProvider"]</strong>.
+                    Please enter an email address for this site below and click the Register button to finish
+                    logging in.
```

Comment:
The text could be less technical, more focussed on the user. Perhaps we can ask UX.

Reply:
I'll ask in slack